src/App.js

```diff
@@ -127,37 +134,41 @@ export default class App extends Component {
         )
     }
 
+    const PrivateRoute = ({component: Component, ...rest }) => {
```

Comment:
I added this part in my PR and created a separate file for the routes. When #36 is merged I will merge this one